Docs/ProjectSpec.md

```diff
@@ -238,6 +238,10 @@ A source can be provided via a string (the path) or an object of the form:
 	- `file`: a file reference with a parent group will be created (Default for files or directories with extensions)
 	- `group`: a group with all it's containing files. (Default for directories without extensions)
 	- `folder`: a folder reference.
+- [ ] **headerVisibility**: **String** The visibility of any headers. This defaults to `project`, but can be either:
+	- `public`
+	- `private`
+	- `project`
```

Comment:
Maybe we can allow upper-case at first letter. `Public` `Private` `Project`

Reply:
Yeah I guess the decoding could lowercase it first. I'd want the documented way to be lowercase though to match the rest of the spec